strangler-infusereo/code/ruby-service.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra'
+require 'json'
+require 'net/http'
+
+set :port, 9494
+
+$users=[{name: 'foo', id: 1}]
```

Comment:
Typically, I'd say use a class variable, as globals are really discouraged in Ruby. However, I'm a little fuzzy on the context in which a request resides.